.github/workflows/merge-bot-pr.yml

```
@@ -0,0 +1,30 @@
+name: Merge bot PR after CI
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+  # checks: read # For private repositories
+  # actions: read # For private repositories
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
+        id: metadata
+        uses: dependabot/fetch-metadata@v1.5.1
+      - uses: actions/checkout@v3
+      - name: Wait other jobs
+        # if: ${{steps.metadata.outputs.update-type != 'version-update:semver-major'}}
+        uses: kachick/wait-other-jobs@v1.2.1
+        timeout-minutes: 30
+        with:
+          github-token: '${{ secrets.GITHUB_TOKEN }}'
+      - name: Approve and merge
+        # if: ${{steps.metadata.outputs.update-type != 'version-update:semver-major'}}
+        run: gh pr review --approve "$PR_URL" && gh pr merge --auto --squash "$PR_URL"
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: '${{secrets.GITHUB_TOKEN}}'
```

```suggestion
          ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

こうすると、マージ後にもCI/CDが走ってデプロイまでされる筈
自分はあまりPATを使わないか、https://github.com/tibdex/github-app-token でワンショットのトークン生成させる事が多いのだけれど、このリポジトリにはすでに設定があるし良いと思います。